### DISCUSSION TITLE
How to include .github/pull_request_template.md in renovate pull requests

### DISCUSSION THREAD

**Discussion by tobiasweibel** [renovatebot/renovate#17888] [Request Help] [answered]
I'm wondering whether there's an option to include the pull request template of a repository in the renovate pull request body. Basically I'd like to keep the default pull request message that renovate generates but I'd like to add e.g. [.github/pull_request_template.md](https://github.com/renovatebot/renovate/blob/266b5e4798e41c85240edfce2da81042b69d0fca/.github/pull_request_template.md) to the end. All that is needed because the process in my company requires that we fill out that template for each and every PR.

It would help if e.g. `prFooter` (or prHeader) option could be set with e.g. `github>renovatebot/.github/pull_request_template.md` or something similar.

**Comment by rarkins** [comment, marked as answer]
Good idea, please create a feature request so we can discuss further. We'll need to consider both the case where the template is in the same repo instead of the org's .github repo. Also whether we could fetch this automatically somehow or better the require manual configuration like you suggest.

**Reply by tobiasweibel** [reply]
See issue #17897
